judgments/tests/utils/assertions.py

```diff
@@ -2,22 +2,55 @@
 Custom assertion helpers for tests.
 """
 
+from lxml import html
 
-def assert_contains_html(response, html):
-    """
-    Asserts that the given HTML is contained within the response content.
 
-    Raises:
-        AssertionError: If the HTML is not found in the response content.
-    """
-    assert html.replace(" ", "").replace("\n", "") in response.content.decode().replace(" ", "").replace("\n", "")
+def parse_html(html_string):
+    return html.fromstring(html_string)
 
 
-def assert_not_contains_html(response, html):
-    """
-    Asserts that the given HTML is not contained within the response content.
+def parse_html_fragment(html_string):
+    return html.fragments_fromstring(html_string)
```

Comment:
Ooh. I did not know about this, very useful. I've always taken to stripping off the outer XML.

Worried about the mismatch in fragment/fragments: do we mean one or many.

Reply:
My thinking here was that we're passing in a fragment, and this function will return a list of fragments, maybe I could rename it something like `parse_html_fragment_as_fragments` ?